web-frontend/src/pages/Home/Home.tsx

```diff
@@ -79,17 +87,27 @@ export default function Home() {
             </div>
           </div>
 
-          <div className="bg-secondary rounded-2xl w-full max-w-lg p-10">
+          <div className="bg-secondary rounded-2xl w-full max-w-lg p-10 flex flex-col justify-between">
             <p className="mas-body text-f-primary mb-6">
               {Intl.t('home.title-account-address')}
             </p>
-            <Clipboard
-              displayedContent={address}
-              rawContent={address}
-              error={Intl.t('errors.no-content-to-copy')}
-              className="flex flex-row items-center mas-body2 justify-between
-              w-full h-12 px-3 rounded bg-primary cursor-pointer"
-            />
+            <div className="flex w-full justify-between items-center">
+              {mns && (
+                <Tag type="default">
+                  <div className="flex items-center gap-2">
+                    <MassaLogo size={24} />
+                    <div>{mns}</div>
+                  </div>
+                </Tag>
+              )}
+              <Clipboard
+                displayedContent={maskAddress(address)}
+                rawContent={address}
+                error={Intl.t('errors.no-content-to-copy')}
+                className="flex flex-row items-center mas-body2 justify-between
+              w-fit h-12 px-3 rounded bg-primary cursor-pointer"
+              />
```

Comment:
maybe we will need a component to show a mns and or a nickname

Reply:
yes that could be interesting

Reply:
https://github.com/massalabs/ui-kit/pull/433